docs/iqe-ee-proxy.md

```diff
@@ -0,0 +1,44 @@
+# Setup for the IQE to Ephemeral Proxy
+
+### This allows a user to run and debug the IQE python tests locally against a deployment on OpenShift
+
+1. Create an iqe_workspace directory  
+2. Copy get-iqe.py to directory:  
+https://insights-qe.pages.redhat.com/iqe-core-docs/tutorial/part3.html#download-get-iqe-py
+3. Run the script  
+`python ./get-iqe.py`
+4. Create a file in the iqe_workspace directory named iqe_env:  
+`export ENV_FOR_DYNACONF=clowder_smoke`  
+`export DYNACONF_IQE_VAULT_OIDC_HEADLESS=true`  
+`export DYNACONF_SETTINGS__pre_check=0`    
+`export NAMESPACE=[current namespace]`    
```

Comment:
We'll know the namespace after the step 9, but we could simply do:
```suggestion
`export NAMESPACE=$(oc project -q)`    
```

Reply:
Tried this. It does not work in PyCharms.